src/libraries/Microsoft.Bcl.TimeProvider/ref/Microsoft.Bcl.TimeProvider.Forwards.cs

```diff
@@ -0,0 +1,5 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.TimeProvider))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Threading.ITimer))]
```

Comment:
What about `TimeProviderTaskExtensions` looks to me like you are exposing this on `netstandard2.0` but not on `$(NetCoreAppCurrent)`?